emission/tests/pipelineTests/TestLabelInferencePipeline.py

```diff
@@ -0,0 +1,85 @@
+# This tests the label inference pipeline. It uses fake data and placeholder inference algorithms
+# and thus intentionally does not test the real inference algorithms themselves.
+import unittest
+import numpy as np
+from logging import config  # epi.run_intake_pipeline needs this to be done here
+import time
+
+import emission.core.wrapper.user as ecwu
+import emission.pipeline.intake_stage as epi
+import emission.pipeline.reset as epr
+import emission.analysis.classification.inference.labels.pipeline as eacilp
+import emission.core.wrapper.labelprediction as ecwl
+import emission.storage.decorations.analysis_timeseries_queries as esda
+import emission.storage.timeseries.timequery as estt
+import emission.storage.decorations.trip_queries as esdt
+
+class TestLabelInferencePipeline(unittest.TestCase):
```

Comment:
Please also add a `main` (similar to the other tests in `emission/tests`) to make it easier to run the tests in isolation. I'm not quite sure how you ran the tests without that.

Reply:
Will do. I ran with `./e-mission-py.bash -m unittest -v emission.tests.pipelineTests.TestLabelInferencePipeline`.